### PR TITLE
Add archive notice.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# This repo is not actively maintained neither, for the bleeding edge support head over to [swift-transformers](https://github.com/huggingface/swift-transformers)!
```
Comment (pcuenca) [inline README.md]:
```suggestion
# This repo is not actively maintained and has been archived. For an in-development replacement, please head over to [swift-transformers](https://github.com/huggingface/swift-transformers)!
```

**Review by pcuenca — APPROVED** [review]
Thank you! I suppose we then need to enable this, right? (I've never done that):

Screenshot 2023-11-24 at 18 22 41

**Comment by Vaibhavs10** [comment]
Yes! That's the button!

**Review by julien-c — APPROVED** [review]
looks good to me

**Pull request merged**